### ISSUE TITLE
Add travel restrictions

### ISSUE THREAD

**Issue by blakeelias** [Pandemic-Game/pandemic-game#61]
Affects importation rate rather than R.
Currently importation rate is 0 -- instead, make it > 0, but reducible by investing in travel restrictions.

**Comment by anamartinovici** [comment]
I understand "travel restrictions" is different from the current option to "close transit". Can you please explain what each of these options do?

**Comment by blakeelias** [comment]
"Close transit" in the current game means closing *internal* transit, while
"travel restrictions" would mean closing *external* transit.
Closing internal transit lowers the R_{effective} (reproductive number
within your region).
Closing external transit would instead lower the importation rate.
These are two different quantities, which show up in the equation:

E[Infected at time (t+1)] = [Infected at time t] * R_{effective} +
(importation rate)

**Comment by anamartinovici** [comment]
I think we need to make this distinction easier to understand. 'External transit' seems easier to get as it refers to entries in the country. But I'm still not sure what counts as 'internal' - is that travel between states, regions, cities, within city?

**Comment by blakeelias** [comment]
> But I'm still not sure what counts as 'internal' - is that travel between states, regions, cities, within city?


Yes, that's right. Agreed, a better term for this distinction could be made.

"Close borders" vs. "Restrict internal travel"?


________________________________
